.github/pull_request_template.md

```diff
@@ -20,7 +20,7 @@
 
 ## :pencil: Checklist
 <!--- Put an `x` in the boxes that apply -->
-- [ ] I ran `./gradlew spotlessApply` before submitting the PR
+- [ ] I formatted the code with the IDE's reformat action (Ctrl + Shift + L/Cmd + Shift + L)
```

Comment:
Wouldn't it be better if we give directions to the UI element ?
IDE shortcuts may not be available if they are used by the system. (I had this issue on KDE)

Reply:
No way to do that without making the entire thing very wordy. I'm trying to keep this as small as it can be. I intentionally used the wording IntelliJ uses so you can just plop this into a search engine and get the right results should there be a case as yours.